assets/constants.ts

```diff
@@ -6993,6 +6993,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "gsoc_M1.2",
```

Comment:
Woah. Why is this making it into the PR?

Please always check your "Files Changed" tab before requesting any reviews, in order to make sure the changes you submit are the ones you actually want to get merged. This is good PR discipline.

Reply:
Sorry, this happened previously as well. I'll just remove it but any idea how this is automatically getting attached?

Reply:
Try checking with @kevintab95 -- per https://github.com/oppia/oppia/blame/develop/assets/constants.ts#L7044 he merged that change 5 days ago.

Reply:
Hi @kshitij01042002, few questions that will help me debug this:
1. When these files were committed, was the local server still active in the background?
2. If not, what method did you use to shut down the server?

Thanks!

Reply:
1. No the local server was not running while commiting these changes.
2. I usually shutdown the server with ctrl+c, so did this time.

FYI these changes occured when I did a commit on my local branch.
Thanks @kevintab95.

Reply:
Hi @kevintab95 these changes always takes place whenever I push changes to github

Reply:
Hi @kshitij01042002, I tried this out but haven't been able to reproduce on my machine. The file constants.ts seems to be reset to the original state after I shutdown the server using the ctrl + c method. The changes however do show up if the server was not shutdown. Could you share the logs on the console corresponding to the shutdown process? This might help us understand what is happening.

Reply:
Another thing I would suggest trying is, start the server in prod mode (i.e. use the "--prod_env" flag), wait for it to open up the splash page and then stop the server. Execute "git status" and see if there are any changes to the "DEV_MODE" constant in constants.ts -- prod mode also involves modifying the constants file temporarily. If there are changes, it probably means there is some issue with shutting down the server and we will need to investigate that.

Reply:
Also, a note for the future, after you have responded to comment(s) and would like someone to take a look at it, please remember to assign them to the PR to avoid delays. The assignee field needs to reflect the individuals who need to take action on the PR. I almost missed this because I wasn't assigned.

Reply:
Hi @kevintab95 I tried to run the server in prod mode and was exited with an error:-
![image](https://github.com/oppia/oppia/assets/91470808/1a00ba74-6176-4315-9013-388042ee07b9)

![image](https://github.com/oppia/oppia/assets/91470808/5cb6a6df-8f5e-4813-9ffa-81c5b380cbe3)

Also after doing this there were changes in index.yaml file and constants.ts file which wasn't reverted.

Reply:
What changes were in the constants file when the server stopped -- when run in prod mode?